kernel/objects/rapl-driver-intel/objects/RaplDriverIntel.cc

```diff
@@ -145,15 +145,15 @@ namespace mythos {
     }
 
     //determine power unit
-    power_units = bits(x86::getMSR(MSR_RAPL_POWER_UNIT),3,0);
+    power_units = static_cast<uint32_t>(bits(x86::getMSR(MSR_RAPL_POWER_UNIT),3,0));
```

Comment:
can be shorter with `uint32_t(bits...)`. Would be nice to make this reusable for other projects by moving it to the CtrlRegs stuff next to the other MSR functions.